config/game_options.txt

```diff
@@ -96,10 +96,12 @@ MASTER_MODE secret
 ## Set to 0 to disable that mode.
 
 PROBABILITY TRAITOR 9
-PROBABILITY CULT 6
-PROBABILITY DYNAMIC 6
+PROBABILITY CULT 9
+PROBABILITY REVOLUTION 8
+PROBABILITY CLOCKCULT 8
```

Comment:
Clockcult 8? Holy shit.

Reply:
Clockcult 8 is _definitely_ not going to get in.

Reply:
Bumping it down to 4, or should I go lower?

Reply:
Clockcult is a game mode that is enjoyable in small numbers, setting it to 4 will still quadruple it.